doc/sphinx-guides/source/developers/tips.rst

```diff
@@ -102,7 +102,7 @@ Deploying With ``asadmin``
 
 Sometimes you want to deploy code without using Netbeans or from the command line on a server you have ssh'ed into.
 
-For the ``asadmin`` commands below, we assume you have already changed directories to ``/usr/local/glassfish4`` or wherever you have installed Glassfish.
+For the ``asadmin`` commands below, we assume you have already changed directories to ``/usr/local/glassfish4/glassfish/bin`` or wherever you have installed Glassfish.
```

Comment:
Looks good.  There are usually two asadmin locations (`/usr/local/glassfish4/bin/asadmin` and `/usr/local/glassfish4/glassfish/bin/asadmin`); but they should be equivalent - so we may not need to mention both.